scripts/install_third_party_libs.py

```diff
@@ -210,6 +219,74 @@ def ensure_system_python_libraries_are_installed(package, version):
     install_backend_python_libs.pip_install_to_system(package, version)
 
 
+def rewrite_android_proto_files():
+    """Edit all android proto files."""
+    # Since there is no simple configuration for imports when using protobuf to
```

Comment:
This comment relates to the entire function, rather than the following line. So, it should probably be part of the docstring of the function instead.

Perhaps try adding a sentence before it explaining where the Android proto files came from in the first place and why we need to edit/rewrite the Android proto files, to make it clear to someone who's not familiar with what you know.

Reply:
Updated with an explanation.

```
def rewrite_android_proto_files():
    """Android proto files are the part of oppia-proto-api-* directory in the
    third_party_libs directory. Currently, proto files generation doesn't
    support importing other proto files from a different directory.
    Because of it, we had to keep all the proto files in the same directory.
    Since there is no simple configuration for imports when using protobuf to
    generate Python files we need to manually fix the imports.
    See: https://github.com/protocolbuffers/protobuf/issues/1491
    """
```